PullRequestCreated/PullRequestCreated.ts

```diff
@@ -43,20 +43,19 @@ class PullRequestCreated extends OctokitAction {
   }
 
   private additionalFields(parameters: IssueParameters): any {
-    let addutinalFields = {};
-    if (parameters?.parent) {
-      addutinalFields = { parent: { key: parameters.parent } };
-    }
+    return { ...this.parseAdditionalFields(), ...parameters }
```

Comment:
Can we embed the `{...}` into the call site? It looks like unneeded indirection now